TESTING.md

```diff
@@ -28,7 +28,7 @@ Most code changes will fall into one of the following categories.
 ### Writing tests for new features
 
 New code should be covered by unit tests. If the code is difficult to test with
-a unit tests then that is a good sign that it should be refactored to make it
+unit tests then that is a good sign that it should be refactored to make it
```

Comment:
I think it might be slightly more readable with a comma after `tests`

```
unit tests, then that is a good sign that it should be refactored to make it
```